### PR TITLE
Add PR template when creating a new rails project

### PR DESCRIPTION
**Pull request by bronzdoc**
- [ ] Bug fix
- [x] Feature
- [ ] Chore

**Description:**
Adds PR template when creating a new rails project

### CODE REVIEW

File: config_files/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Please include a summary of the change and which issue is fixed or which feature is introduced. If changes to the behaviour are made, clearly describe what changes.
+
+**Screenshots:**
+
+If changes to the UI are made, please include screenshots of the before and after. 
+
+**Related story:**
+
+Link to the relevant story in Pivotal Tracker.
```
Comment (arielj) [inline config_files/pull_request_template.md:19]: Here we could say `relevant story in Pivotal Tracker or issue in GitHub`

**Review by arielj — APPROVED** [review]
LGTM! just added one small comment but not a blocker or anything

File: config_files/pull_request_template.md (hunk 1 above)
Comment (bronzdoc) [reply config_files/pull_request_template.md:19]: @arielj the title should also be something like: `Related story/Github Issue:`?

Comment (arielj) [reply config_files/pull_request_template.md:19]: Yes sure, to make it more generic.

**Pull request merged**